common/common_helper.sh

```diff
@@ -800,3 +800,23 @@ function kruize_local_disable() {
 	fi
 }
 
+function kruize_remote_patch() {
+	CRC_DIR="./manifests/crc/default-db-included-installation"
+	KRUIZE_CRC_DEPLOY_MANIFEST_OPENSHIFT="${CRC_DIR}/openshift/kruize-crc-openshift.yaml"
+	KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE="${CRC_DIR}/minikube/kruize-crc-minikube.yaml"
+
+
+  if [ ${CLUSTER_TYPE} == "minikube" ]; then
+    if grep -q '"isROSEnabled": "false"' ${KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE}; then
+      echo "match found"
+      sed -i 's/"isROSEnabled": "false"/"isROSEnabled": "true"/' ${KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE}
+    else
+      echo "Error: Match not found" >&2
+      exit 1
+    fi
+  elif [ ${CLUSTER_TYPE} == "openshift" ]; then
+    sed -i 's/"isROSEnabled": "false"/"isROSEnabled": "true"/' ${KRUIZE_CRC_DEPLOY_MANIFEST_OPENSHIFT}
```

Comment:
@msvinaykumar You will need to update this change in kruize_local_disable method. And remove local flag updation from it.

Reply:
Can you please run the demo and paste the output.

Reply:
```
Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T08:00:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T08:15:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T08:30:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T08:45:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T09:00:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T09:15:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T09:30:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T09:45:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T10:00:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T10:15:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T10:30:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T10:45:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T11:00:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T11:15:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T11:30:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T11:45:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T12:00:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T12:15:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T12:30:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T12:45:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T13:00:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=rhsso-operator|deployment|sso&interval_end_time=2023-03-16T13:15:00.000Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

************************************************************
[{'version': 'v2.0', 'experiment_name': 'quarkus-resteasy-kruize-min-http-response-time-db_0', 'cluster_name': 'cluster-one-division-bell', 'performance_profile': 'resource-optimization-openshift', 'mode': 'monitor', 'target_cluster': 'remote', 'kubernetes_objects': [{'type': 'deployment', 'name': 'tfb-qrh-deployment_0', 'namespace': 'default_0', 'containers': [{'container_image_name': 'kruize/tfb-db:1.15', 'container_name': 'tfb-server-0'}, {'container_image_name': 'kruize/tfb-qrh:1.13.2.F_et17', 'container_name': 'tfb-server-1'}]}], 'trial_settings': {'measurement_duration': '15min'}, 'recommendation_settings': {'threshold': '0.1'}}]

************************************************************

Validating the input json...

Creating the experiment...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/createExperiment
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Experiment registered successfully with Kruize. View registered experiments at /listExperiments","httpcode":201,"documentationLink":"","status":"SUCCESS"}
Experiment name =  quarkus-resteasy-kruize-min-http-response-time-db_0
Deployment name =  tfb-qrh-deployment_0
Namespace =  default_0

*************************************************************************************
Updating results for one of the experiments and fetching recommendations from Kruize...
*************************************************************************************


Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=quarkus-resteasy-kruize-min-http-response-time-db_0&interval_end_time=2023-04-02T01:00:00.110Z
Response status code =  201

Updating the results...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateResults
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  201
{"message":"Results added successfully! View saved results at /listExperiments.","httpcode":201,"documentationLink":"","status":"SUCCESS"}

Updating the Recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/updateRecommendations?experiment_name=quarkus-resteasy-kruize-min-http-response-time-db_0&interval_end_time=2023-04-02T13:30:00.680Z
Response status code =  201

Listing the recommendations...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listRecommendations?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Listing the experiments...
URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:32760/listExperiments?rm=true
KRUIZE UI URL =  http://worker-2.kruize.lab.psi.pnq2.redhat.com:31908
Response status code =  200

Success! Monitoring demo setup took 306 seconds

```

Reply:
> @msvinaykumar You will need to update this change in kruize_local_disable method. And remove local flag updation from it.

Done